cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCtx.java

```diff
@@ -367,6 +372,24 @@ public AlgorithmID getKdf() {
 		return context_id;
 	}
 
+	/**
+	 * Controls whether or not to include the Context ID in messages generated from this context
```

Comment:
using a dot (".") at the end of the first sentence create nicer javadoc.

Reply:
Fixed in commit 5db48d3f1f225e0dcb3588042c0ed19c5f863d04